src/assets/custom/css/_sass/_training-card.scss

```
@@ -0,0 +1,77 @@
+.training-card {
+  display: flex;
+  flex-direction: column;
+}
+
+.training-card__primary-content {
+  background-color: $slate;
+  padding-bottom: 40px;
+  padding-left: $lateral-space--medium;
+  padding-right: $lateral-space--medium;
+  padding-top: 35px;
+
+  .training-card--featured & {
+    padding-top: 20px;
+  }
+}
+
+.training-card__secondary-content {
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  justify-content: space-between;
+  padding: 20px $lateral-space--medium 40px;
+
+  @include medium-large-and-extra-large{
+    align-items: flex-start;
+  }
+
+  .training-card--featured & {
+    background-color: $slate-light;
+  }
+}
+
+.training-card__avatar {
+  width: 40px;
+  margin-right: 15px;
+}
+
+.training-card__tag {
+  @include ponti-bold;
+  align-items: center;
+  color: white;
+  display: flex;
+  flex-direction: row;
+}
+
+.training-card__title {
+  color: white;
+
+  @include small {
+    @include mercer-bold;
+  }
+  @include medium-large-and-extra-large {
+    @include freed-bold;
+  }
+
+  .training-card--featured & {
+    @include freed-bold;
+  }
```

Personally I find this slightly faster to read and it also avoids overrides in the CSS

Not a big deal, just an idea

```suggestion
  .training-card:not(.training-card--featured) & {
    @include small {
      @include mercer-bold;
    }
    @include medium-large-and-extra-large {
      @include freed-bold;
    }
  }

  .training-card--featured & {
    @include freed-bold;
  }
```